core/pom.xml

```diff
@@ -118,6 +118,23 @@
       <groupId>org.apache.zookeeper</groupId>
       <artifactId>zookeeper</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.jsonwebtoken</groupId>
+      <artifactId>jjwt-api</artifactId>
+      <version>0.12.6</version>
+    </dependency>
```

Comment:
If users don't use the JWSFilter feature, we still need to include this new dependency?

Reply:
Yes for now. Of course, we can make this as a `profile`.